opm/input/eclipse/EclipseState/Util/OrderedMap.hpp

```diff
@@ -25,9 +25,59 @@
 #include <string>
 #include <stdexcept>
 #include <iterator>
+#include <string>
```

Comment:
We already include `<string>` three lines earlier.  Maybe we should sort the headers and remove duplicates too?